Take02/wwwroot/js/external.js

```diff
@@ -5,6 +5,21 @@
         url: url,
         method: "GET"
     }).success(function (partialView) {
-        $('#newIngredient').append(partialView);
+        // Set the sort order based on existing items on the page
+        var maxNumber = 0;
+        $('.ingredientsContainer').find('input.number').each(function() {
```

Comment:
This is a goofy construction for sure, but it's kinda just the way things go with javascript at times...